nix/plugin-overlay.nix

```diff
@@ -0,0 +1,9 @@
+{
+  name,
+  self,
+}: final: prev: {
+  vimPlugins.rocks-nvim = final.pkgs.vimUtils.buildVimPluginFrom2Nix {
```

Comment:
careful, in latest nixpkgs, it's now `buildVimPlugin` ;)

Reply:
Hmm, maybe we should use `buildLuarocksPackage` instead.
It's gonna need toml-edit and some other dependencies.